demo/demo-schema/src/main/resources/microservices/pojo/server.yaml

```diff
@@ -150,7 +150,7 @@ paths:
       parameters:
         - name: TestRequest
           in: body
-          required: true
+          required: false
```

Comment:
We must not change a test case if it is required.  Please add a new one if possible.

Reply:
here the test case is to check null when no body , but if we configure required as true it will always throw exception in client itself as null body not allowed as per swagger. so changed this to false as that is what is expected out of this test case.